engines/router/missionctl/log/resultlog/console.go

```diff
@@ -15,16 +15,20 @@ func newConsoleLogger() *ConsoleLogger {
 // write logs the given TuringResultLogEntry to the console
 func (*ConsoleLogger) write(turLogEntry *TuringResultLogEntry) error {
 	// Get context-specific logger
-	logger := log.WithContext(*turLogEntry.ctx)
-	// Add request and responses
+	logger := log.Glob()
```

Comment:
context is no longer stored in the `TuringResultLogEntry`. The Turing Request ID that was being added with the contextual logger is now directly stored as a field in this struct.